### PR TITLE
Data transfer   back

### PR DESCRIPTION
**Pull request by rmangin-fr**
Do not merge this !
Should I copy OCPPDataTransferStatus stuff into OCPPClient ?

### CODE REVIEW

File: src/client/ocpi/OCPIChargingStationClient.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
...
+import { OCPPDataTransferRequest, OCPPDataTransferResponse } from '../../types/ocpp/OCPPServer';
```
Comment (rmangin-fr) [inline src/client/ocpi/OCPIChargingStationClient.ts:2]: @LucasBrazi06: I need your advise, I'm not sure if I should copy definitions into the Client file ?

Comment (ClaudeROSSI) [reply src/client/ocpi/OCPIChargingStationClient.ts:2]: as discussed, please mimic what is done for other the commands. Dedicated interfaces have to be defined

**Review by ClaudeROSSI — CHANGES_REQUESTED** [review]
as discussed, please mimic the logic of other commands

**Comment by rmangin-fr** [comment]
Need to test on QA

**Review by LucasBrazi06 — APPROVED** [review]
GOOD JOB!!!

**Pull request merged**